src/ray/common/scheduling/resource_instance_set.cc

```diff
@@ -253,7 +394,27 @@ std::string NodeResourceInstanceSet::DebugString() const {
     first = false;
     buffer << id.Binary() << ": " << FixedPointVectorToString(quantity);
   }
-  buffer << "}";
+  buffer << "}, {";
+
+  first = true;
+  for (const auto &[original_id, indexed_ids] : pg_indexed_resources_) {
```

Comment:
I used it when testing the change in this PR to make sure the information in the auxiliary map is correct . But makes sense, it might not be interesting after the feature is fully tested and stable.